hello_world/react-client/src/components/Hello.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import HelloTransaction from '../transactions/hello_transaction';
+import { APIClient } from '@liskhq/lisk-api-client';
+import * as cryptography from '@liskhq/lisk-cryptography';
+
+const networkIdentifier = cryptography.getNetworkIdentifier(
+    "23ce0366ef0a14a91e5fd4b1591fc880ffbef9d988ff8bebf8f3666b0c09597d",
+    "Lisk",
+);
+
+const API_BASEURL = 'http://localhost:4000';
```

Comment:
It would be nice to define the identifier and API_BASEURL in one place, e.g. config.js file and import it from all the files that use it. So that if needed, it can be easily changed just in one place.